test/provider/expect.unittest.ts

```diff
@@ -238,13 +238,81 @@ describe('When creating a non-directory FDC3 app', () => {
     });
 });
 
+describe('When creating an external connection', () => {
+    // External connections won't have a corresponding window. These "windows" MUST have a createdTime of undefined to produce accurate scenerios.
+    // Connections from external windows must connect within the WINDOW_EXPECT_TO_CREATED timeout, which is a much stricter requirement than normal OF windows
```

Comment:
Hmm, don't hate me, but did I get this wrong? Maybe we do want to give a more generous `WINDOW_EXPECT_TO_CREATED ` timeout. (Also, should the name of `WINDOW_EXPECT_TO_CREATED` change?

Reply:
Ahh, yeah - should rename.

Should maybe increase a little - but depends to what extent. It's currently 200ms for the window creation, followed by 5000ms for the subsequent IAB connection. Bumping the former to 500-1000 ish seems fine, if it's any longer than that then it seems a bit pointless having the two distinct timeouts.

Reply:
Nudge on this, doesn't seem to have changed